### PR TITLE
done

### PR DESCRIPTION
**Pull request by panianchuk**
https://panianchuk.github.io/react_people-table-basics/#/

### CODE REVIEW

File: src/components/PeoplePage/PeoplePage.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      setIsLoading(true);
+      setIsPersonsLoaded(false);
+      setErrorNoteShown(false);
+
+      try {
+        const person = await getPeople();
+
+        setPersons(person);
+      } catch {
+        setIsLoading(false);
```
Comment (slavanosatskyi) [inline src/components/PeoplePage/PeoplePage.tsx:29]: you don't need to duplicate it here as long as it is in finally block

File: src/components/PeoplePage/PeoplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { useMatch } from 'react-router-dom';
+import { PeopleTable } from '../PeopleTable';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+export const PeoplePage = () => {
+  const [persons, setPersons] = useState<Person[]>([]);
```
Comment (slavanosatskyi) [inline src/components/PeoplePage/PeoplePage.tsx:10]:
```suggestion
  const [peolple, setPeolple] = useState<Person[]>([]);
```

File: src/components/PeoplePage/PeoplePage.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  const [isPersonsLoaded, setIsPersonsLoaded] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isErrorNoteShown, setErrorNoteShown] = useState(false);
+  const match = useMatch('/people/:slug');
+  const selectedPerson = match?.params.slug;
+  const isCountPersonsLoaded = persons.length === 0;
```
Comment (slavanosatskyi) [inline src/components/PeoplePage/PeoplePage.tsx:16]: the name gives wrong association. Better to name in a way that shows your intention - isPeopleListEmpty or something

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
Nice, though I would change a couple of variables names

**Review by ViktorHavryliuk — APPROVED** [review]

**Pull request closed without merging**